homeassistant/components/monarchmoney/sensor.py

```diff
@@ -0,0 +1,201 @@
+"""Sensor config - monarch money."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+
+from typedmonarchmoney.models import MonarchAccount, MonarchCashflowSummary
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import CURRENCY_DOLLAR, PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import MonarchMoneyConfigEntry
+from .entity import MonarchMoneyAccountEntity, MonarchMoneyCashFlowEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class MonarchMoneySensorEntityDescription(SensorEntityDescription):
+    """Base common class."""
+
+
+@dataclass(frozen=True, kw_only=True)
+class MonarchMoneyAccountSensorEntityDescription(MonarchMoneySensorEntityDescription):
+    """Describe an account sensor entity."""
+
+    value_fn: Callable[[MonarchAccount], StateType | datetime]
+    picture_fn: Callable[[MonarchAccount], str | None] | None = None
+
+
+@dataclass(frozen=True, kw_only=True)
+class MonarchMoneyCashflowSensorEntityDescription(MonarchMoneySensorEntityDescription):
+    """Describe a cashflow sensor entity."""
+
+    summary_fn: Callable[[MonarchCashflowSummary], StateType] | None
```

Comment:
```suggestion
    summary_fn: Callable[[MonarchCashflowSummary], StateType]
```
This way you can remove the None check